### PR TITLE
Csharp5 Standard Markdown Fixes (Struts)

### PR DESCRIPTION
**Pull request by cmcclister**
Per https://ceapex.visualstudio.com/CPS/_workitems/edit/169416

@BillWagner please review and let me know if I can refine this further. Thanks!

### CODE REVIEW

File: spec/types.md [hunk 1]
```diff
@@ -2,385 +2,418 @@
 
 ## General
```
Comment (BillWagner) [inline spec/types.md]: This file was already covered by the previous PR. Are there new changes?

File: spec/structs.md [hunk 2]
```diff
@@ -2,376 +2,440 @@
...
 ### Automatically implemented properties
 
-Automatically implemented properties (§15.7.4) use hidden backing fields, which are only accessible to the property accessors. \[*Note*: This access restriction means that constructors in structs containing automatically implemented properties often need an explicit constructor initializer where they would not otherwise need one, to satisfy the requirement of all fields being definitely assigned before any function member is invoked or the constructor returns. *end note*\]
+Automatically implemented properties (§15.7.4) use hidden backing fields, which are only accessible to the property accessors. 
 
-[[]{#_Ref451675312 .anchor}]{#_Toc445783066 .anchor}
+> [!NOTE]
+> [*Note*: This access restriction means that constructors in `structs` containing automatically implemented properties often need an explicit constructor initializer where they would not otherwise need one, to satisfy the requirement of all fields being definitely assigned before any function member is invoked or the constructor returns. *end note*]
 
+[_Ref451675312](#_Toc445783066)
```
Comment (BillWagner) [inline spec/structs.md]: That line can be removed.

File: spec/structs.md [hunk 3]
```diff
@@ -2,376 +2,440 @@
...
 
-A *struct-declaration* is a *type-declaration* (§14.7) that declares a new struct:
+A *struct-declaration* is a *type-declaration* (§14.7) that declares a new `struct`:
 
-[]{#Grammar_struct_declaration .anchor}struct-declaration:\
-attributes~opt~ struct-modifiers~opt~ *partial*~opt~ *struct* identifier type-parameter-list~opt\
-~struct-interfaces~opt~ type-parameter-constraints-clauses~opt~ struct-body *;*~opt~
+[#Grammar_struct_declaration](struct-declaration)
+```ANTLR
+struct-declaration:
```
Comment (BillWagner) [inline spec/structs.md]: Not your edit, but somehow this grammar production really got mangled in the markdown production. Lines 20 - 22 should be replaced with the following: 
```suggestion
struct_declaration
    : attributes? struct_modifier* 'partial'? 'struct' identifier type_parameter_list?
      struct_interfaces? type_parameter_constraints_clause* struct_body ';'?
    ;
```

File: spec/types.md (hunk 1 above)
Comment (cmcclister) [reply spec/types.md]: Sorry @BillWagner no new changes here. I'll refresh my fork moving forward. Thanks!

File: spec/structs.md (hunk 2 above)
Comment (cmcclister) [reply spec/structs.md]: @BillWagner that line has been removed.

File: spec/structs.md (hunk 3 above)
Comment (cmcclister) [reply spec/structs.md]: @BillWagner this ANTLR code has been updated.

**Comment by cmcclister** [comment]
@BillWagner unsafe.md has also been formatted.

**Comment by cmcclister** [comment]
@BillWagner sorry I didn't see your feedback prior to adding Variables. That is my last inclusion in this PR. I will work on the outstanding issues in the Unsafe file.

**Comment by BillWagner** [comment]
This is ready now. Thanks for the additional updates @cmcclister 
I'll :shipit: now

/cc @SyntaxC4

**Pull request merged**